po/fr.po

```diff
@@ -1,5 +1,723 @@
 msgid ""
 msgstr ""
+"Project-Id-Version: \n"
+"POT-Creation-Date: 2020-10-07 21:17+0200\n"
+"PO-Revision-Date: 2020-10-13 22:04+0200\n"
+"Last-Translator: Yannicka\n"
+"Language-Team: \n"
+"Language: fr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
-"Plural-Forms: nplurals=2; plural=n>1;\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.4.1\n"
+"X-Poedit-Basepath: .\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Poedit-SearchPath-0: .\n"
+
+#: src/Dialogs/ExportDialog.vala:151
+msgid "Export to:"
+msgstr "Exporter vers :"
+
+#: src/Dialogs/ExportDialog.vala:157
+msgid "Select Folder"
+msgstr "Sélectionner un dossier"
+
+#: src/Dialogs/ExportDialog.vala:166
+msgid "Format:"
+msgstr "Format :"
+
+#: src/Dialogs/ExportDialog.vala:179
+msgid "Quality:"
+msgstr "Qualité :"
+
+#: src/Dialogs/ExportDialog.vala:191
+msgid "Compression:"
+msgstr "Compression :"
+
+#: src/Dialogs/ExportDialog.vala:205
+msgid "Transparency:"
+msgstr "Transparence :"
+
+#: src/Dialogs/ExportDialog.vala:218
+msgid "Scale:"
+msgstr "Échelle :"
+
+#: src/Dialogs/ExportDialog.vala:242 src/FileFormat/FileManager.vala:49
+#: src/FileFormat/FileManager.vala:105
+msgid "Cancel"
+msgstr "Annuler"
+
+#: src/Dialogs/ExportDialog.vala:249 src/Dialogs/ShortcutsDialog.vala:63
+#: src/Layouts/HeaderBar.vala:112
+msgid "Export"
+msgstr "Exporter"
+
+#: src/Dialogs/ReleaseDialog.vala:47 src/Dialogs/SettingsDialog.vala:217
+msgid ""
+"WARNING!\n"
+"Akira is still under development and not ready for production. Missing "
+"features, random bugs, and black holes opening in your kitchen are to be "
+"expected."
+msgstr ""
+"ATTENTION !\n"
+"Akira est encore en cours de développement et n'est pas prêt à être mis en "
+"production. Attendez-vous à ce que des fonctionnalités manquent, des bugs "
+"se produisent voire qu'un trou noir s'ouvre dans votre cuisine."
```

Comment:
Maybe `n'est pas prêt à être utilisé` (not sure if we should use "utilisé" as a project, or "utilisée" as an application)  would make more sense, as this application is not deployed. The meaning is more "not ready to be used as an end user application".

> "ATTENTION !\n"
"Akira est encore en cours de développement et n'est pas prêt à être utilisé."
"Attendez-vous à ce que des fonctionnalités manquent, des bugs "
"se produisent voire qu'un trou noir s'ouvre dans votre cuisine."